front/src/js/paint/structDate/vector2.ts

```diff
@@ -63,21 +86,31 @@ export class Vector2 {
     divisionOnNumber(num: number, type: string = 'double') {
         this.x /= num;
         this.y /= num;
-
+        this.accuracy = this.accuracy / num;
```

Comment:
?

Reply:
Yeah, my bad. I didn't understand accuracy destination at that moment